pkgs/top-level/python-packages.nix

```diff
@@ -31828,6 +31828,28 @@ EOF
     };
   };
 
+  rss2email = buildPythonPackage rec {
```

Comment:
this is an application and not a Python library and thus doesn't belong in `python-packages.nix`.